### PR TITLE
Implement dataloader adapters

### PR DESCRIPTION
**Pull request by saturn4er**

### CODE REVIEW

File: generator/plugins/dataloader/config.go [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+
 type DataLoadersConfig struct {
 	OutputPath string `mapstructure:"output_path"`
 }
+
 type ProviderConfig struct {
-	Name           string `mapstructure:"name"`
-	WaitDurationMs int    `mapstructure:"wait_duration_ms"`
+	Name         string        `mapstructure:"name"`
+	WaitDuration time.Duration `mapstructure:"wait_duration_ms"`
```
Comment (damour) [inline generator/plugins/dataloader/config.go]: mapstructure:"wait_duration_ms" => mapstructure:"wait_duration"

File: generator/plugins/proto2gql/dataloaders.go [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+		return errors.Wrap(err, "failed to resolve file type file")
+	}
+
+	dataLoaderOutType, err := g.TypeOutputTypeResolver(outputMsgTypeFile, resultMessage)
+	if err != nil {
+		return errors.Wrap(err, "failed to resolve output type")
+	}
+
+	var fetchCode func(importer *importer.Importer) string
+	if cfg.Type == DataLoaderType1ToN {
```
Comment (damour) [inline generator/plugins/proto2gql/dataloaders.go]: Maybe we should add cfg.Type validation?

Comment (saturn4er) [reply generator/plugins/proto2gql/dataloaders.go]: ready

Comment (saturn4er) [reply generator/plugins/proto2gql/dataloaders.go]: I validate it earlier

File: generator/plugins/dataloader/config.go (hunk 1 above)
Comment (saturn4er) [reply generator/plugins/dataloader/config.go]: ready

**Pull request merged**